src/MICore/CommandFactories/gdb.cs

```diff
@@ -194,5 +199,16 @@ public override Task EnableTargetAsyncOption()
             // that isn't actually supported by gdb. 
             await _debugger.CmdAsync("kill", ResultClass.None);
         }
+        public override async Task Signal(string sig)
+        {
+            string command = String.Format("-interpreter-exec console \"signal {0}\"", sig);
+            await _debugger.CmdAsync(command, ResultClass.running);
```

Comment:
Do you mean the 'signal' GDB commands outputs nothing? If so, sounds okay I suppose.

---
In reply to: [58890158](https://github.com/Microsoft/MIEngine/pull/259#discussion_r58890158) [](ancestors = 58890158)